midi_app_controller/gui/midi_status.py

```diff
@@ -49,8 +51,22 @@ def increase_opacity(ll: LayerList):
 for action in SLIDER_ACTIONS:
     get_app().register_action(action)
 
-# TODO Get actions directly from app-model when it's supported.
-NAPARI_ACTIONS = HELP_ACTIONS + LAYER_ACTIONS + VIEW_ACTIONS + SLIDER_ACTIONS
+
+def get_actions() -> List[CommandRule]:
+    """Returns a list of all actions currently registered in app model (and available in the command pallette)."""
+    return sorted(
+        list(
+            set(
+                item.command
+                for item in get_app().menus.get_menu(MenusRegistry.COMMAND_PALETTE_ID)
+                if isinstance(item, MenuItem)
+            )
+        ),
+        key=lambda command: command.id,
+    )
+
+
+NAPARI_ACTIONS = get_actions()
 
 state_manager = StateManager(NAPARI_ACTIONS, get_app())
```

Comment:
We want to support actions added during runtime and I think that this PR is a good place to do it. So maybe `get_actions` should be passed to the `StateManager` instead of `NAPARI_ACTIONS`?